test/system/000-setup.bats

```diff
@@ -19,4 +19,6 @@ load 'libs/helpers'
         _pull_and_cache_distro_image fedora "$((system_version-1))" || false
         _pull_and_cache_distro_image fedora "$((system_version-2))" || false
     fi
+
+    _setup_docker_registry
```

Comment:
Are we setting up our local Docker registry and tearing it down for every test case?  Even the ones that don't need it?

If so, then we should probably not do that because setting up the registry is quite expensive.  It involves generating generating cryptographic certificates, which is CPU intensive.  We are downloading the `quay.io/toolbox_tests/registry` image, and it doesn't look like we are caching that one, but even if it did the local disk I/O would still be measurable.  Then actualling running the container for the registry and adding an image to it, which both involve disk I/O.

Reply:
No, `000-setup.bats` is a test file that is called before all other (considering the whole directory is used for testing or the file is hand-picked). The registry is set up only once for the whole duration of the system tests.